src/server/serverWrappers.cpp

```diff
@@ -354,8 +355,9 @@ void processClientUsername(const int sock, const char *buff, std::pair<const int
     strncpy(outBuff + TCP_HEADER_SIZE + 1, client.second.entry.username, NAMELEN);
 
     gm->createMarine(client.first);
-    gm->getMarine(client.first).first.setPosition(100, yPos);
-    yPos += 150;
+    auto& marine = gm->getMarine(client.first).first;
+    marine.setPosition(100, yPos);
+    yPos += marine.getW() * 10;
```

Comment:
You literally told me to change it from 150 to 10x the player width
Make up your mind